ndk/src/relay/subscriptions.ts

```diff
@@ -2,13 +2,13 @@ import { EventEmitter } from "tseep";
 import type { Sub, SubscriptionOptions } from "nostr-tools";
```

Comment:
This file is still using the old stuff that doesn't exist anymore. Still needs to be fixed.